website/pages/docs/nia/installation/configuration.mdx

```diff
@@ -227,7 +227,7 @@ task {
 
 ### Multiple Provider Configurations
 
-Consul-Terraform-Sync supports the [Terraform feature to define multiple configurations](https://www.terraform.io/docs/configuration/providers.html#alias-multiple-provider-configurations) for the same provider by utilizing the `alias` meta-argument. Define multiple provider blocks with the same provider name and set the `alias` to a unique value across a given provider. Select which provider configuration to use for a task by specifying the configuration with the provider name and alias (`<name>.<alias>`) within the list of providers in the [`task.provider`](#task) parameter.
+Consul-Terraform-Sync supports the [Terraform feature to define multiple configurations](https://www.terraform.io/docs/configuration/providers.html#alias-multiple-provider-configurations) for the same provider by utilizing the `alias` meta-argument. Define multiple provider blocks with the same provider name and set the `alias` to a unique value across a given provider. Select which provider configuration to use for a task by specifying the configuration with the provider name and alias (`<name>.<alias>`) within the list of providers in the [`task.provider`](#task) parameter. Although multiple different providers are allowed per task, only one provider instance of a type of provider is allowed per task.
```

Comment:
This was difficult to digest for me, a lot of qualifiers to keep straight: "multiple different" "per" "only one ... type", "per". I do like the goal of extra clarity here, however could you rephrase it?

Took a jab, feel free to take or another angle!

> A task can use multiple providers, but only one provider instance of a provider is allowed per task

Reply:
Agree, I didn't love how it turned out after a couple read. I like your suggestion, thank you!